cmd/kube-dns/app/server.go

```diff
@@ -142,6 +148,11 @@ func (d *KubeDNSServer) startSkyDNSServer() {
 		Domain:  d.domain,
 		DnsAddr: fmt.Sprintf("%s:%d", d.dnsBindAddress, d.dnsPort),
 	}
+	if d.nameServers != "" {
+		for _, hostPort := range strings.Split(d.nameServers, ",") {
```

Comment:
I took the variable name from the skydns code. I guess hostPort means a host and a port. Happy to change the name though. I'll update the pull request tomorrow morning.